regen-network/regen-web
# fix: Swap out icons on profile tabs, in user "..." menus, and "not found" icons

## Description

Closes: regen-network/regen-registry#1165



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test

Testing both light and dark mode:
- Go to https://deploy-preview-2005--regen-registry.netlify.app/ and login:
   - with an account that is an issuer with no project/batch yet to see the updated tabs icons as well as the icons shown when there's no project or batch yet
   - with an account with no bridgable ecocredits to see the icon show when there's no bridgable ecocredits
- You can also use https://deploy-preview-2005--regen-registry.netlify.app/ecocredits/accounts/regen1gzw5snmgee76r83cxumdgpsn2zht3l7cycu7en/portfolio to see the icons shown when there's no ecocredits, no basket tokens, no retirement certificates and no bridged ecocredits

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
